extensions/wikia/AdEngine/js/lookup/prebid/prebidHelper.js

```diff
@@ -0,0 +1,34 @@
+define('ext.wikia.adEngine.lookup.prebid.prebidHelper', function() {
+	var adUnits = [];
+
+	function getAdUnits(adapter, slots) {
+		var adaptersAdUnits = [];
+
+		Object.keys(slots).forEach(function(slotName) {
+			adaptersAdUnits.push(adapter.prepareAdUnit(slotName, slots[slotName]));
```

Comment:
What do you think about passing skin name instead of slots config? Then it looks like we can get rid off `getSlots` method